core/web/resolver/spec.go

```diff
@@ -771,12 +771,15 @@ func (b *BlockhashStoreSpecResolver) EVMChainID() *string {
 }
 
 // FromAddress returns the job's FromAddress param, if any.
-func (b *BlockhashStoreSpecResolver) FromAddress() *string {
-	if b.spec.FromAddress == nil {
+func (b *BlockhashStoreSpecResolver) FromAddresses() *[]string {
```

Comment:
Pointer to a slice seems odd, but I guess there are ten other instances in here :shrug:.